bears/vcs/git/GitCommitBear.py

```diff
@@ -20,6 +22,12 @@ class GitCommitBear(GlobalBear):
     LICENSE = 'AGPL-3.0'
     ASCIINEMA_URL = 'https://asciinema.org/a/e146c9739ojhr8396wedsvf0d'
     CAN_DETECT = {'Formatting'}
+    SUPPORTED_HOSTS = {
+        'github': (r'(?:[Cc]lose[sd]?|[Rr]esolve[sd]?'
+                   '|[Ff]ix(?:e[sd]?)?)\s+'),
+        'gitlab': (r'(?:[Cc]los(?:e[sd]?|ing)|[Rr]esolv(?:e[sd]?|ing)'
+                   '|[Ff]ix(?:e[sd]|ing)?)\s+')
```

Comment:
ah okay, cool
btw: I wouldn't enclose in `(?:...)`, this shall handle the regex inside the run-function later^^ it's redundant information